MvvmCross.iOS.Support.JASidePanels/MvxSidePanelsPresenter.cs

```diff
@@ -146,29 +154,42 @@ private void ProcessActivePanelPresentation(MvxPresentationHint hint)
         /// <param name="hint">The hint.</param>
         private void ProcessPopToRootPresentation(MvxPresentationHint hint)
         {
-            var popHint = hint as MvxPanelPopToRootPresentationHint;
-            if (popHint != null)
+            try
             {
-                var panelHint = popHint;
-
-                switch (panelHint.Panel)
+                var popHint = hint as MvxPanelPopToRootPresentationHint;
+                if (popHint != null)
                 {
-                    case MvxPanelEnum.Center:
-                        if (CentrePanelUiNavigationController() != null)
-                            CentrePanelUiNavigationController().PopToRootViewController(false);
-                        break;
+                    var panelHint = popHint;
 
-                    case MvxPanelEnum.Left:
-                        if (LeftPanelUiNavigationController() != null)
-                            LeftPanelUiNavigationController().PopToRootViewController(false);
-                        break;
+                    switch (panelHint.Panel)
+                    {
+                        case MvxPanelEnum.Center:
+                            if (CentrePanelUiNavigationController() != null)
+                            {
+                                CentrePanelUiNavigationController().PopToRootViewController(false);
+                            }
+                            break;
 
-                    case MvxPanelEnum.Right:
-                        if (RightPanelUiNavigationController() != null)
-                            RightPanelUiNavigationController().PopToRootViewController(false);
-                        break;
+                        case MvxPanelEnum.Left:
+                            if (LeftPanelUiNavigationController() != null)
+                            {
+                                LeftPanelUiNavigationController().PopToRootViewController(false);
+                            }
+                            break;
+
+                        case MvxPanelEnum.Right:
+                            if (RightPanelUiNavigationController() != null)
+                            {
+                                RightPanelUiNavigationController().PopToRootViewController(false);
+                            }
+                            break;
+                    }
                 }
             }
+            catch (Exception ex)
+            {
+                
```

Comment:
it helps us for when we fix the NsInteral Inconsistency errors. Does that bother you?